core/src/miner/mem_pool.rs

```diff
@@ -901,6 +907,16 @@ impl MemPool {
                 }
                 true
             })
+            .filter(|t| {
+                let mut result = true;
```

Comment:
You can change it to `range.contains(t.inserted_timestamp)` if you use Range.

Reply:
Thank you

Reply:
range.contains is nightly-only experimental API. What is the best way to mimic the behavior?

Reply:
Oops. I didn't know that contains is not stable yet.
I think the below expression is simplest and most efficient in the current stable.
```rust
range.min() <= Some(t.inserted_timestamp) && Some(t.inserted_timestamp) <= range.max()
```